meetings/2022-03-07.md

```diff
@@ -0,0 +1,132 @@
+* eP: I'll merge the PR now
+
+### Simplify Solid-OIDC Conformance Discovery language
+
+[PR#89](https://github.com/solid/solid-oidc/pull/89)
+
+* eP: submitted ealier this morning
+* AC: I will review after the meeting
+* NAS: seems like a good clarification
+
+ACTION: AC to review and merge
+
+### make id token verification not be exclusive to AS
+
+[PR#90](https://github.com/solid/solid-oidc/pull/90)
+
+* eP: this PR was last minute
+* eP: (screen sharing to show preview)
+* eP: Issuer discovery section separated from verification
+* eP: the use of `solid:oidcIssuer` predicate made explicit
+* NAS: to clarify: we added language about link headers. Is it possible to publish link headers while the WebID Profile has no `solid:oidcIssuer` triples?
+* eP: the link header section says that _if_ the values are present in the headers, the values MUST be the same as in the RDF. IOW, it would be an invalid WebID Profile (for Solid-OIDC) to have an inconsistency between the headers and the RDF document in this regard
+* NAS: thank you
+* eP: Sub-section for ID token validation, reference to OIDC-Core (e.g. MUST be valid according to parent spec). Verifying party must perform OIDC discovery
+* eP: this decouples this section from the authorization server
+* eP: this allows the text to reference the specific verification sections, delegating to parent specs, as needed
+* AC: Looks like a good direction, especially being able to separate sections and refer to them as needed. It should help with defining conformance classes.
+* eP: I will plan to start adding conformance classes (possibly next PR). E.g., add "client may perform client validation"
+
+### Accessing NonRDF-Sources directly via browser
+
+[Issue#31](https://github.com/solid/solid-oidc/issues/31)
+
+* eP: old issue (Dec 2019) that has been discussed a few times. There have been experiments with cookies and origin restrictions
+* eP: this is the case where the RS acts as a client (RP)
+* eP: if you access the Solid Storage directly, there would be no use of an access token. Just an ID token.
+* eP: this is an example where the client must validate the ID token
+* eP: it would be good to add a note to the spec related to this use case. For example, a way to login and act as a client.
+* AC: This has come up couple of times, we implemented it and subsequently removed the implementation. At high level I would argue that it doesn't belong in Solid-OIDC spec itself. However it would belong in the primer and/or a best practices document. One of the important things about Solid is that one can separate apps from the data; here we would tie your app to the data. We want to be clear what that means and how that would work in the context of Solid. We also need to specify the implications: an implementation would need to be careful how client identifiers are used. Also how other apps could try to use cookies to circumvent any client identifier restrictions.
```

Comment:
```suggestion
* AC: This has come up couple of times. We implemented it and subsequently removed the implementation. At high level, I would argue that it doesn't belong in Solid-OIDC spec itself. However, it would belong in the primer and/or a best practices document. One of the important things about Solid is that one can separate apps from the data; here we would tie your app to the data. We want to be clear what that means and how that would work in the context of Solid. We also need to specify the implications: an implementation would need to be careful how client identifiers are used. Also how other apps could try to use cookies to circumvent any client identifier restrictions.
```